opennms-dao/pom.xml

```diff
@@ -53,6 +53,16 @@
       <groupId>org.opennms</groupId>
       <artifactId>opennms-config-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.opennms.features.config</groupId>
+      <artifactId>org.opennms.features.config.upgrade</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.opennms.features.config.service</groupId>
+      <artifactId>org.opennms.features.config.service.api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
org.opennms.features.config.service.api is included in CM-Phase1 already.

Reply:
I am not sure I understand?